### PR TITLE
Init

### PR DESCRIPTION
**Pull request by delvedor**
I've added some basic files to start this project.
We can write here or open specific issues to discuss.

cc @fastify/fastify

### CODE REVIEW

**Review by mcollina — APPROVED** [review]
LGTM

**Comment by allevo** [comment]
Maybe use `fastify/fastify-env` is convenient :wink:

And `fastify-swagger`

File: test/basic.test.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
+'use strict'
+
+const { test, beforeEach, afterEach } = require('tap')
+const server = require('../server')
+var fastify = null
+
+beforeEach(done => {
+  server.build({
+    'log-level': 'silent'
```
Comment (mcollina) [inline test/basic.test.js:9]: are you sure that this is not `logLevel`?

File: test/basic.test.js [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+  }, (err, instance) => {
+    if (err) throw err
```
Comment (mcollina) [inline test/basic.test.js:11]: this is bad practice

File: server.js [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+const Fastify = require('fastify')
+
+function build (opts, callback) {
+  const fastify = Fastify({
+    level: opts['log-level'] || (process.env.NODE_ENV === 'production' ? 'error' : 'info')
+  })
+
+  fastify.get('/status', (req, reply) => {
+    reply.send({ status: 'ok' })
+  })
```
Comment (mcollina) [inline server.js:13]: we should show a plugin somewhere

File: server.js [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+
+  fastify.ready(err => {
+    callback(err, fastify, opts)
+  })
```
Comment (mcollina) [inline server.js:17]: there is no need to call the callback within `ready`, avvio serializes everything.

**Review by mcollina — COMMENTED** [review]
We need to put a bit more work on the project structure.

**Comment by Uzlopak** [comment]
closing due to inactivity

**Pull request closed without merging**